entrypoint.sh

```diff
@@ -1,59 +0,0 @@
-#!/bin/bash
-
-: ${SSH_KEY:=$HOME/ssh-private-rsa.key}
-
-# When mounting a volume; docker ( <= v1.10.3) does not provide means to set permissions on the mount point
-# "setup-workspace" subcommand is a workaround for that
-if [[ "$1" != setup-workspace && ( \
-    ! -w $HOME || \
-    ! -d $HOME/.ssh || \
-    ( -f "${SSH_KEY}" && ! -f $HOME/.ssh/id_rsa )
-) ]]; then
-    # we need root privileges to change permission of the directory
-    if ! sudo "$0" setup-workspace; then
-        echo setup-workspace failed\; giving up.;
-        exit 1;
-    fi
-fi
-
-case "$1" in
-    setup-workspace)
-        if [ ! -w $HOME ]; then
-            # change permission; not recursive by purpose
-            # recursive chown can take ages if the volume has a lot of files and directories
-            exec chown migrator:migrator $HOME;
-        fi
-
-        if [ ! -d $HOME/.ssh ]; then
-            mkdir $HOME/.ssh &&
-                chmod go-rwx $HOME/.ssh &&
-                chown migrator:migrator $HOME/.ssh;
-        fi
-
-        if [[ -f "${SSH_KEY}" && ! -f $HOME/.ssh/id_rsa ]]; then
-            cp "${SSH_KEY}" $HOME/.ssh/id_rsa &&
-                chown migrator:migrator $HOME/.ssh/id_rsa &&
-                chmod go-rwx $HOME/.ssh/id_rsa;
-        fi
-        ;;
-
-    wait-up)
```

Comment:
Migrating a small project will be quick and in most cases this docker image can be run as a "one-shot" command. In cases where the project is huge, and if it has to run over a long period of time, say over a weekend or something, then it will be handy to be able to exec into a running container and setup the migration, with the help of `screen` and then detach from it.

Thats the way it was originally envisaged. Now that was with the limited understanding of docker and its way of working back then. Very likely that it is not right approach in docker land.

Obviously those aspects are the concern of the host machine; but what if it doesnt come with the right tools for it; i.e if it is CoreOS, which doesn't come with screen.

Only way I can see this new form working is if it is possible to `docker run` this image from within another container that has the right tools and has access to the host's docker daemon.

As we saw on Friday SZ had trouble with the long running migration. Would have helped if it was possible to run from within LDC.